units/crons.go

```diff
@@ -320,6 +320,25 @@ func PopulateLastContainerFinishTimeJobs() amboy.QueueOperation {
 	}
 }
 
+func PopulateParentDecommissionJobs() amboy.QueueOperation {
+	return func(queue amboy.Queue) error {
+		catcher := grip.NewBasicCatcher()
+		ts := util.RoundPartOfHour(1).Format(tsFormat)
```

Comment:
just checking: this will run once an hour? that seems too infrequent, but we can change.